From 4095ee95141063ff3f91153e84bd029b019b8dad Mon Sep 17 00:00:00 2001
From: Romain Naour <romain.naour@openwide.fr>
Date: Fri, 26 Dec 2014 17:44:15 +0100
Subject: [PATCH] Makefile.am: remove unsafe headers path

/usr/local/include is probably not needed and it's unsafe for
cross-compilation.

Signed-off-by: Romain Naour <romain.naour@openwide.fr>
---
 Makefile.am | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index a9ad5b9..0109e5d 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -38,12 +38,12 @@ else
 CFLAGS += -O2
 endif
 if IS_GCC4
-CPPFLAGS += -iquote. -I/usr/local/include
+CPPFLAGS += -iquote.
 if IS_DEBUG
 CFLAGS += -Wno-pointer-sign
 endif
 else
-CPPFLAGS += -I. -I- -I/usr/local/include
+CPPFLAGS += -I. -I-
 endif
 endif
 
-- 
1.9.3

